examples/snake.rs

```diff
@@ -0,0 +1,324 @@
+extern crate coffee;
+
+use coffee::graphics::{
+    Color, Font, Frame, Mesh, Point, Rectangle, Shape, Text, Window, WindowSettings,
+};
+use coffee::input::keyboard::KeyCode;
+use coffee::input::{self, keyboard, Input};
+
+use coffee::load::Task;
+use coffee::{Game, Timer};
+
+use rand::seq::IteratorRandom;
+
+fn main() {
+    SnakeGame::run(WindowSettings {
+        title: String::from("Snake"),
+        size: (900, 600),
+        resizable: false,
+        maximized: false,
+        fullscreen: false,
+    })
+    .expect("An error occured while starting the game");
+}
+
+fn new_random_pos() -> (f32, f32) {
+    let mut rng = rand::thread_rng();
+    let x = (0..900).step_by(30).choose(&mut rng).unwrap() as f32;
+    let y = (0..600).step_by(30).choose(&mut rng).unwrap() as f32;
```

Comment:
May be a good idea to define a couple of constants for the window dimensions.

Reply:
I'll reply only one time for all the change request, but they're all good ideas. I'll try to make these change tonight or tommorow. There's just a problem about Point. I tried to use at first but I had errors about alias used as constructor in some case. So idk if I should use Point where I can use it and use Position the rest of the time or just keeper with Position everywhere.